packages/autorest.typescript/test/rlcIntegration/generated/bodyFormDataRest/src/bodyFormData.ts

```diff
@@ -21,6 +22,9 @@ export default function createClient(
     ...options,
     userAgentOptions: {
       userAgentPrefix
+    },
+    loggingOptions: {
+      logger: logger.info
```

Comment:
My understanding is the customer could feed their own logger method, if that's undefined we would use default one. It would be like:

```typescript
logger: options.logger || logger.info
```